addons/stock/models/stock_move_line.py

```diff
@@ -860,3 +860,61 @@ def action_open_reference(self):
             'views': [[False, "form"]],
             'res_id': self.id,
         }
+
+    def _get_revert_inventory_move_values(self):
+        """ Called when user manually revert inventory
+        just before creating the corresponding stock move.
+
+        :return: dict with all values needed to create a new `stock.move` with its move line.
```

Comment:
This docstring doesn't really add any useful info and is too restrictive in its description (maybe we will use this method again in later business logic).

```suggestion
    def _get_revert_inventory_move_values(self):
```